src/Workspaces/Core/Portable/CodeFixes/CodeFixContext.cs

```diff
@@ -15,33 +16,27 @@ namespace Microsoft.CodeAnalysis.CodeFixes
     /// <summary>
     /// Context for code fixes provided by a <see cref="CodeFixProvider"/>.
     /// </summary>
-    public struct CodeFixContext : ITypeScriptCodeFixContext
+    public readonly struct CodeFixContext : ITypeScriptCodeFixContext
```

Comment:
```suggestion
    public readonly struct CodeFixContext
```

Types outside the external access layer are not allowed to implement types from the external access layer.

Reply:
Right now TypeScript is casting the CodeFixContext to the interface directly, so can't remove until we update TS. The goal of this change is to remove this interface entirely.